hack/python-sdk-v2/gen-sdk.sh

```diff
@@ -35,7 +35,7 @@ if [[ ! -f "$SWAGGER_CODEGEN_JAR" ]]; then
 fi
 
 echo "Generating Python SDK for Training Operator V2 ..."
-java -jar "${SWAGGER_CODEGEN_JAR}" generate -i "${SWAGGER_CODEGEN_FILE}" -g python -o "${SDK_OUTPUT_PATH}" -c "${SWAGGER_CODEGEN_CONF}" -p=packageVersion="${SDK_VERSION}"
+java -jar "${SWAGGER_CODEGEN_JAR}" generate -i "${SWAGGER_CODEGEN_FILE}" -g python --global-property apiTests=false,modelTests=false -o "${SDK_OUTPUT_PATH}" -c "${SWAGGER_CODEGEN_CONF}" -p=packageVersion="${SDK_VERSION}"
```

Comment:
Please add the TODO statement, similar as I did in my PR: https://github.com/kubeflow/training-operator/pull/2324/files#diff-37b22ae163a534b2b732fb386e8dc7e7d7a996141258d2567465019ad7139982R38-R42.

Reply:
Thanks, I wasn't sure if you guys wanted to keep that TODO. Updated!